mlrun/api/utils/auth/verifier.py

```diff
@@ -205,6 +205,9 @@ def authenticate_request(
             if "x-data-session-override" in request.headers:
                 auth_info.data_session = request.headers["x-data-session-override"]
 
+            if auth_info.user_unix_id is None:
```

Comment:
Add a comment explaining that before Iguazio version X the user unix id was not returned in the headers of the auth response so we have to request it explicitly